### PR TITLE
POC of Mesh Lambda Deployment

### PR DESCRIPTION
**Pull request by joeds13**

### CODE REVIEW

File: terraform/mesh_lambdas/locals.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+locals {
+  name = "${var.name_prefix}-mesh"
+
+  mesh_url = {
+    integration = "https://msg.intspineservices.nhs.uk/"
+    production  = "https://mesh-sync.spineservices.nhs.uk"
```
Comment (ranisen) [inline terraform/mesh_lambdas/locals.tf]: production should be https://msg.spineservices.nhs.uk

File: terraform/mesh_lambdas/ssm_config.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
+resource "aws_ssm_parameter" "ca_cert" {
+  name  = "/${local.name}/mesh/MESH_CA_CERT"
+  type  = "String"
```
Comment (ranisen) [inline terraform/mesh_lambdas/ssm_config.tf]: Secrets should be secure strings?

File: terraform/mesh_lambdas/locals.tf (hunk 1 above)
Comment (joeds13) [reply terraform/mesh_lambdas/locals.tf]: Updated

But maybe https://digital.nhs.uk/developer/api-catalogue/message-exchange-for-social-care-and-health-api#api-description__environments-and-testing needs updating too?

File: terraform/mesh_lambdas/ssm_config.tf (hunk 2 above)
Comment (joeds13) [reply terraform/mesh_lambdas/ssm_config.tf]: Yep, will be once I have done the secrets pattern, these are just placeholder resources

**Pull request closed without merging**